### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by NJaryal**
Home Task1
1) Header
2) Footer
3) Courses
  - course component
  - courses list
  - courses filter

### CODE REVIEW

File: src/app/common/models/courses.model.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export class Courses {
+  Id: string;
+  Title: string;
```
Comment (IlyaLisitsin) [inline src/app/common/models/courses.model.ts]: Please, don't use capital letters in naming of keys, especially in combination with regular letters. Everything should be consistent.

File: angular.json [hunk 2]
```diff
@@ -1,124 +0,0 @@
...
-          "configurations": {
-            "production": {
-              "devServerTarget": "courses-app:serve:production"
-            }
-          }
-        }
-      }
-    }},
-  "defaultProject": "courses-app"
-}
```
Comment (IlyaLisitsin) [inline angular.json]: I'm pretty sure, that we don't need to delete "angular.json" file, as it is the main config of the application :)

Comment (NJaryal) [reply angular.json]: OMG - It was certainly a mistake i suppose.

File: src/app/common/models/courses.model.ts (hunk 1 above)
Comment (NJaryal) [reply src/app/common/models/courses.model.ts]: Updated

**Pull request merged**